src/transaction-path/constants.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  DELIMITER: ':',
+
+  // This is in fact a single backslash character, but in stupid JS it's two
+  // because you're escaping a JS escape character \.
```

Comment:
I think this is relevant to basically any language without built-in support for raw strings, hence I'd delete the lines 3-5.

Reply:
Addressed in https://github.com/apiaryio/dredd-transactions/pull/126/commits/150d0cb002cc3ee2cbfe0dd54050f4d43453391d.